docs/user-guide/integrations.md

```diff
@@ -5,7 +5,7 @@
 * Sublime Text 3:
     * [SublimeLinter-eslint](https://github.com/roadhump/SublimeLinter-eslint)
     * [Build Next](https://github.com/albertosantini/sublimetext-buildnext)
-* [Vim](https://github.com/vim-syntastic/syntastic/tree/master/syntax_checkers/javascript)
+* Vim: [ALE](https://github.com/w0rp/ale) and [Syntastic](https://github.com/vim-syntastic/syntastic/tree/master/syntax_checkers/javascript) support ESlint
```

Comment:
Would it make more sense to have these be a sub-list, similar to what is done in Sublime Text 3 just above this entry?

Reply:
I think that does make a little more sense. I went ahead and amended that.